examples/jenkins/jenkins-persistent-template.json

```diff
@@ -204,11 +195,35 @@
       "value": "password"
     },
     {
+      "name": "MEMORY_REQUEST",
+      "displayName": "Memory Lower Limit",
+      "description": "Reserved amount of memory for the container.",
+      "value": "1200Mi"
+    },
+    {
+      "name": "MEMORY_LIMIT",
+      "displayName": "Memory Upper Limit",
+      "description": "Maximum amount of memory the container can use.",
+      "value": "2000Mi"
+    },
```

Comment:
you are quota'd against your upper limit.  if you have a 2gig quota, you can only deploy resources with a total of 2 gigs of upper limit.  So if a user deploy this, they would not be able to deploy anything else in online.  

We need to keep this limit to 512meg.  It's been validated to work with our jenkins image, if you're trying to use another image, you shouldn't really be using this template anyway. This template defines env vars that are respected specifically by our image.

Would like to know why you're using the dockerhub jenkins image instead of our image?